### DISCUSSION TITLE
Crash at Launch (Linux AppImage)

### DISCUSSION THREAD

**Discussion by chill0r** [ppy/osu#16592] [Q&A] [answered]
When launching osu!lazer AppImage (2022.118.0) the application crashes with
`Unhandled exception. Realms.Exceptions.RealmException: Provided schema version 12 is less than last set version 13.`
(see osu_stderr.log for full stderr output).
Runtime.log attached too but no errors logged there.
No other logfiles created under ~/.local/share/osu/logs

OS: Debian 11 (bullseye)
osu! Version: AppImage 2022.118.0

Logfiles:
[runtime.log](https://github.com/ppy/osu/files/7925040/runtime.log)
[osu_stderr.log](https://github.com/ppy/osu/files/7925041/osu_stderr.log)

**Comment by peppy** [comment, marked as answer]
Have you run a newer release by self-compiling? If so, you'll need to wait for the next actual release, or delete `client.realm`. Note if you delete that file you will lose all beatmaps/scores/settings.

**Comment by chill0r** [comment]
Thanks for the reply. 
Yeah I compiled and ran the current master branch beforehand.
Deleting the `~/.local/share/osu/client.realm` fixed the AppImage crash.
